src/index.html

```diff
@@ -1,14 +1,37 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="navigation__wraper">
+        <div class="navigation__logo">
+          <img src="./images/logo.png" alt="logo">
+        </div>
+        <nav class="navigation__nav">
+          <ul class="navigation__list">
+            <li class="navigation__item"><a href="#" class="navigation__link is-active">Apple</a></li>
```

Comment:
```suggestion
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a href="#" class="nav__link is-active">
                Apple
              </a>
            </li>
```
You can simplify the class names. Also, pay attention to the formatting
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/ce639341-b586-45a1-adae-f5afe3ab9622)